.github/workflows/stale-issue.yml

```diff
@@ -0,0 +1,55 @@
+# GitHub action based on https://github.com/actions/stale
+
+name: 'Close stale issues and PRs'
+on:
+  # !!! To be removed when merged to master
+  workflow_dispatch:
+  schedule:
+    # Execute every day 4:00 UTC
+    - cron: '0 4 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          # GLOBAL ------------------------------------------------------------
+          # !!! Debug is on until stabilized
+          debug-only: true
```

Comment:
Does that mean it won't actually change anything?

Reply:
> Run the stale workflow as dry-run.
> No GitHub API calls that can alter your issues and pull requests will happen.
> Useful to debug or when you want to configure the stale workflow safely.

I wonder how we can know if it's work or not, by examining the action output log maybe?

Reply:
Yes, you can see the same output, just modifications are not applied.
See: https://github.com/gradle/gradle-private/actions/runs/2902491778